RecoLocalTracker/Phase2TrackerRecHits/interface/Phase2StripCPEDummy.h

```diff
@@ -0,0 +1,20 @@
+#ifndef RecoLocalTracker_Phase2TrackerRecHits_Phase2StripCPEDummy_H
+#define RecoLocalTracker_Phase2TrackerRecHits_Phase2StripCPEDummy_H
+
+#include "RecoLocalTracker/ClusterParameterEstimator/interface/ClusterParameterEstimator.h"
+#include "Geometry/CommonDetUnit/interface/GeomDetUnit.h"
+#include "DataFormats/Phase2TrackerCluster/interface/Phase2TrackerCluster1D.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+
+class Phase2StripCPEDummy : public ClusterParameterEstimator<Phase2TrackerCluster1D> {
```

Comment:
maybe Phase2StripTrivialCPE
instead of "Dummy"?
Trivial appears to be more commonly used for something producing results with very simple logic;
a Dummy is more for something that does nothing at all.